src/commands/moderation/prefixCommands/functions/deleteVersion.ts

```diff
@@ -0,0 +1,149 @@
+import { ChatInputCommandInteraction, Colors, TextChannel, User } from 'discord.js';
+import { constantsConfig, getConn, PrefixCommandVersion, Logger, makeEmbed, PrefixCommandChannelDefaultVersion, clearSinglePrefixCommandVersionCache, PrefixCommand, refreshSinglePrefixCommandCache, clearSinglePrefixCommandChannelDefaultVersionCache } from '../../../../lib';
+
+const noConnEmbed = makeEmbed({
+    title: 'Prefix Commands - Delete Version - No Connection',
+    description: 'Could not connect to the database. Unable to delete the prefix command version.',
+    color: Colors.Red,
+});
+
+const contentPresentEmbed = makeEmbed({
+    title: 'Prefix Commands - Delete Version - Content Present',
+    description: 'There is content present for this command version. Please delete the content first, or use the `force` option to delete the command version and all the command contents for the version.',
+    color: Colors.Red,
+});
+
+const channelDefaultVersionPresentEmbed = makeEmbed({
+    title: 'Prefix Commands - Delete Version - Default Channel Versions Present',
+    description: 'There is one or more channel with this version selected as its default version. Please change or unset the default version for those channels first, or use the `force` option to delete the command version and all the default channel versions referencing it (making them default back to the GENERIC version).',
+    color: Colors.Red,
+});
+
+const failedEmbed = (versionId: string) => makeEmbed({
+    title: 'Prefix Commands - Delete Version - Failed',
+    description: `Failed to delete the prefix command version with id ${versionId}.`,
+    color: Colors.Red,
+});
+
+const doesNotExistsEmbed = (version: string) => makeEmbed({
+    title: 'Prefix Commands - Delete Version - Does not exist',
+    description: `The prefix command version ${version} does not exists. Can not delete it.`,
```

Comment:
```suggestion
    description: `The prefix command version ${version} does not exists. Cannot delete it.`,
```
"Cannot" is generally more common. I suggest we use that but obviously that's personal preference and I'm happy to leave it as is!